DevEdu.API/DevEdu.DAL/Repositories/IStudentAnswerOnTaskRepository.cs

```diff
@@ -5,12 +5,12 @@ namespace DevEdu.DAL.Repositories
 {
     public interface IStudentAnswerOnTaskRepository
     {
-        void AddStudentAnswerOnTask(StudentAnswerOnTaskDto studentResponse);
-        void DeleteStudentAnswerOnTask(int taskId, int studentId);
-        List<StudentAnswerOnTaskDto> GetAllStudentAnswerOnTask();
-        List<StudentAnswerOnTaskDto> GetStudentAnswerByTaskIdAndStudentIdOnTask(StudentAnswerOnTaskDto studentResponse);
-        void UpdateStatusAnswerOnTask(int taskId, int studentId, int statusId);
-        void UpdateStudentAnswerOnTask(StudentAnswerOnTaskDto studentResponse);
+        void AddStudentAnswerOnTask(StudentAnswerOnTaskDto dto);
+        void DeleteStudentAnswerOnTask(StudentAnswerOnTaskDto dto);
+        List<StudentAnswerOnTaskDto> GetAllStudentAnswersOnTask();
+        StudentAnswerOnTaskDto GetStudentAnswerOnTaskByTaskIdAndStudentId(StudentAnswerOnTaskDto dto);
+        void ChangeStatusOfStudentAnswerOnTask(StudentAnswerOnTaskDto dto, int statusId);
```

Comment:
please change to void ChangeStatusOfStudentAnswerOnTask(StudentAnswerOnTaskDto dto) and do all the logic in service